### DISCUSSION TITLE
Adding disk to NSM issues

### DISCUSSION THREAD

**Discussion by Hammer-Bear** [Security-Onion-Solutions/securityonion#8668] [Unsupported Versions] [answered]
So I'm trying to increase the storage of one of our search nodes and I'm going down the LVM approach as per https://docs.securityonion.net/en/2.3/new-disk.html as NSM is a separate logical volume and when trying to create a new volume on the new disk with `pvcreate /dev/sdc` i get `Device /dev/sdc excluded by a filter.` . I've already had a look around and most suggest unmounting it etc but it's a fresh disk with nothing on it. Am I missing something or is there something specific to SO as I've used LVM before and not come across this issue?

**Comment by reyesj2** [comment, marked as answer]
Try:
```
wipefs -a /dev/sdc
```

Then try to recreate the physical volume

**Reply by Hammer-Bear** [reply]
Thanks, i had to do some messing around in `fdisk` but eventually sorted it and `wipefs` worked and so did `pvcreate`.
